doc/htmldoc/installation/index.rst

```diff
@@ -29,8 +29,16 @@ Install NEST
 
        I'm a developer who wants to :ref:`do development in NEST <dev_install>`
 
+|
+
+Install NESTML + NEST
+---------------------
+
+|nestml| I'm a user who wants to :ref:`create or customize models <nestml>`
```

Comment:
```suggestion
|nestml| I'm a user who wants to :ref:`create or customize neuron and synapse models <nestml>`
```

Reply:
The "|nestml|" part does not seem to render well on RtD. Suggest to link directly to https://nestml.readthedocs.org/

Reply:
Suggest to remove "custom models" from the left-hand side menu as it links outside of the NEST RtD documentation which can be confusing.

Reply:
Ok I will directy link to nestml.
On an aside, do you have information for installing nestml with nest? we have docker that contains nestml and nest, but if you install the two separately how does a user ensure they work together? This is something not for this PR, but we should make sure it's clear to users.


I think we should  keep custom models: this was something discussed with Jochen  as well, and wanting to highlight nestml more. I looked to find a way to include an icon for external links using the new theme but couldn't find anything yet.

Reply:
see here for current install page of nest: https://nest-test.readthedocs.io/en/add-intro-models/installation/index.html